crates/ruff_cli/src/commands/snapshots/ruff_cli__commands__run__test__E902_E902.py.snap

```diff
@@ -1,7 +1,7 @@
 ---
 source: crates/ruff_cli/src/commands/run.rs
 ---
-/home/ferris/project/code.py:1:1: E902 Permission denied (os error 13)
-/home/ferris/project/notebook.ipynb:1:1: E902 Permission denied (os error 13)
-/home/ferris/project/pyproject.toml:1:1: E902 Permission denied (os error 13)
+[1m/home/ferris/project/code.py[0m[36m:[0m1[36m:[0m1[36m:[0m [1;31mE902[0m Permission denied (os error 13)
```

Comment:
Can we remove the unicode colors before printing to text?

Reply:
yes this needs to be fixed